android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java

```diff
@@ -335,58 +327,59 @@ private void updatePreferences() {
             }
         }
         removePreferenceIfPresent(PREF_SYNC_AND_SERVICES_LINK);
-        String fingerprinting = BravePrefServiceBridge.getInstance().getFingerprintingControlType();
-        String cosmeticFiltering =
-                BravePrefServiceBridge.getInstance().getCosmeticFilteringControlType();
-        String cookiesBlockType = BravePrefServiceBridge.getInstance().getCookiesBlockType();
+
         String getNoScriptControlType =
                 BravePrefServiceBridge.getInstance().getNoScriptControlType();
 
         SharedPreferences sharedPreferences = ContextUtils.getAppSharedPreferences();
 
+        int blockAdTrackersPref = sharedPreferences.getInt(PREF_BLOCK_TRACKERS_ADS, 1);
+        int cookiesBlockPref = sharedPreferences.getInt(PREF_BLOCK_CROSS_SITE_COOKIES, 1);
+        int fingerprintingPref = sharedPreferences.getInt(PREF_FINGERPRINTING_PROTECTION, 1);
+
         if (getNoScriptControlType.equals(BraveShieldsContentSettings.BLOCK_RESOURCE)) {
             mBlockScriptsPref.setChecked(true);
         } else {
             mBlockScriptsPref.setChecked(false);
         }
 
-        if (cosmeticFiltering.equals(BraveShieldsContentSettings.BLOCK_THIRDPARTY_RESOURCE)) {
+        if (blockAdTrackersPref == 0) {
```

Comment:
It's not obvious what these options (0, 1, 2) really mean. We should probably explicitly define them.

Reply:
not sure if duplicates, but they use they use the code in there to hook up to the browser implementation